Gemfile

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+source 'https://rubygems.org'
+
+gem 'byebug'
+gem 'pry-byebug'
+
+group 'deployment' do
+  gem 'dlss-capistrano', require: false
```

Comment:
This is not a requirement of this repo. Each individual repo may have a different version of this.

Reply:
I figured we would deploy THIS repo, too.

Reply:
easiest way to keep it up to date is to treat it like all the other repos, no?